documentation/sphinx/source/release-notes.rst

```diff
@@ -7,15 +7,18 @@ Release Notes
 
 Features
 --------
+* Improved team collection for data distribution that builds a balanced number of teams per server and gurantees that each server has at least one team. `(PR #1785) <https://github.com/apple/foundationdb/pull/1785>`_.
 
 Performance
 -----------
 
 Fixes
 -----
 
+* Set the priority of redundant teams to remove as PRIORITY_TEAM_REDUNDANT, instead of PRIORITY_TEAM_UNHEALTHY. `(PR #1802 <https://github.com/apple/foundationdb/pull/1802>)`_.
```

Comment:
```suggestion
* Set the priority of redundant teams to remove as PRIORITY_TEAM_REDUNDANT, instead of PRIORITY_TEAM_UNHEALTHY. `(PR #1802) <https://github.com/apple/foundationdb/pull/1802>)`_.

```